developer-guide/concepts-and-designs/multicluster/architecture.md

```diff
@@ -0,0 +1,54 @@
+# Multiple Cluster Architecture
+
+## Introduction
+
+Today, it's very common for organizations to run a handful of Kubernetes Clusters. Each kubernetes cluster is a relatively self-contained unit. And there are several teams working on tying all clusters, such as kubefed. More and more KubeSphere users and customers have expressed their kneed interests in managing all clusters through KubeSphere.
+
+## Summary
+For KubeSphere users and customers, the two highest priority ones are relate to Operating clusters in one place and High Availability.
```

Comment:
the two highest priority ones are relate to Operating clusters in one place and High Availability. => the two highest priority ones are operating clusters in one place and providing high availability at the same time.

Reply:
changed as suggested